docs/running-locally.md

```diff
@@ -32,6 +32,9 @@ Note:
 * For **Windows WSL2**
     * Configure [`.wslconfig`](https://docs.microsoft.com/en-us/windows/wsl/wsl-config#configuration-setting-for-wslconfig) to limit memory usage by the WSL2 to prevent VSCode OOM.
 
+* For **Windows File System**
+    * Make sure git symbolic link is set to True when cloning the repo, clone the repo with `git clone -c core.symlinks=true core.autocrlf=false <URL>`
```

Comment:
This is fairly redundant as written with "when cloning the repo, clone the repo with". The description also restates the command in less clear terms, IMO

```suggestion
    * Clone the repo with `git clone -c core.symlinks=true core.autocrlf=false https://github.com/<my-fork>/argo-workflows.git`
```

- simple and direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- specify most of the URL